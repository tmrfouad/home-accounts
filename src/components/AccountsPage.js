import React from 'react';
import AccountList from './AccountList';

const AccountsPage = () => (
  <div>
    <AccountList />
  </div>
);

export default AccountsPage;
