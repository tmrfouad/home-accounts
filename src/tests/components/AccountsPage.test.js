import React from 'react';
import { shallow } from 'enzyme';
import AccountsPage from '../../components/AccountsPage';

test('should render AccountsPage correctly', () => {
  const wrapper = shallow(<AccountsPage />);
  expect(wrapper).toMatchSnapshot();
});
