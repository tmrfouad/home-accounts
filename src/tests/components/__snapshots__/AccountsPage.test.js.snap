// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render AccountsPage correctly 1`] = `
<div>
  <Connect(AccountList) />
</div>
`;
